### DISCUSSION TITLE
Expo web vs Next

### DISCUSSION THREAD

**Discussion by simonhoss** [tamagui/tamagui#64] [Q&A]
Hey

Thank you for this nice library. I wonder why not using expo web instead of next? I can only imagine that Next is more optimized for web builds? Is that the case?

Thanks for a response.

Simon

**Comment by natew** [comment]
Next is more optimized for sure, it supports SSR. I'm also just not familiar with Expo web.

**Reply by simonhoss** [reply]
Yes. SSR might be a killer argument for some scenarios. Thanks for your response
